app/experimenter/experiments/constants.py

```diff
@@ -390,6 +398,9 @@ class EmailType(models.TextChoices):
     # Serializer validation errors
     ERROR_DUPLICATE_BRANCH_NAME = "Branch names must be unique."
     ERROR_SINGLE_BRANCH_FOR_ROLLOUT = "A rollout may have only a single reference branch"
+    ERROR_ROLLOUT_VERSION_SUPPORT = (
+        "Rollouts are not supported for the given application and version number."
```

Comment:
This is just an improvement suggestion that QA pointed me out in the earlier PR to mention the version no. in the errors messages like rollouts are supported for this and this version for this particular application. So that when we use this error message in the frontend we can help the users which versions it clearly supports 💪

Reply:
that's a great suggestion! I'll bring this up with jared when we dig into [which applications and versions](https://github.com/mozilla/experimenter/pull/7554#discussion_r937178467) we want to support

Reply:
perfect 🎉 🎉

Reply:
Great suggestion @yashika-khurana 🙏